### PR TITLE
gcs: add launcher

### PR DESCRIPTION
**Pull request by luke7637**
Add a launcher to launch the `autorccar_gcs` package using the `ros2 launch` command.

The command to launch autorccar_gcs is as follows:
```
ros2 launch autorccar_gcs autorccar_gcs.launch.py
```

### CODE REVIEW

File: ros2/src/autorccar_gcs/launch/autorccar_gcs.launch.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+def generate_launch_description():
+
+    planning_control_node = Node(
+        package="autorccar_gcs",
+        executable="autorccar_gcs",
+        name="autorccar_gcs",
+        parameters=[],
+        output="screen",
+    )
```
Comment (dlfksj) [inline ros2/src/autorccar_gcs/launch/autorccar_gcs.launch.py]: planning_control_node 에서 gcs_node로 이름 변경

Comment (dlfksj) [reply ros2/src/autorccar_gcs/launch/autorccar_gcs.launch.py]:
```suggestion
def generate_launch_description():

    gcs_node = Node(
        package="autorccar_gcs",
        executable="autorccar_gcs",
        name="autorccar_gcs",
        parameters=[],
        output="screen",
    )
```

File: ros2/src/autorccar_gcs/launch/autorccar_gcs.launch.py [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+    ld = LaunchDescription()
+    ld.add_action(planning_control_node)
```
Comment (dlfksj) [inline ros2/src/autorccar_gcs/launch/autorccar_gcs.launch.py]:
```suggestion
    ld.add_action(gcs_node)
```

**Review by dlfksj — APPROVED** [review]

**Review by gspark87 — APPROVED** [review]
LGTM

**Pull request merged**